public/lang/pl.js

```diff
@@ -98,7 +98,7 @@ export default {
         position: "Pozycja",
         chunk: "Chunk",
         region: {
-            region: "Region",
```

Comment:
I think the word `region` is more correct, since the minecraft world contains regions, not territories (*tereny*).